apps/server/src/app/auth/utils/jwt-auth.strategy.ts

```diff
@@ -14,8 +15,13 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
     });
   }
 
-  async validate(payload: any) {
-    return true;
-    return { userId: payload.sub, username: payload.username };
+  async validate(payload: User & { exp: number, iat: number }): Promise<User> {
+    // this validates the user by default should we call the auth service to do that ?
+    const { exp, iat, ...user } = payload
+    if (iat < exp) {
```

Comment:
please use proper jwt check:
1. check if the signature valid
2. time is >= nbf (not before timestamp)
3. time is <= exp (expire timestamp)

iat is the issued at timestamp